test/runtests.jl

```diff
@@ -60,4 +60,12 @@ facts("Listables") do
     @fact_throws MethodError sin()
 end
 
+facts("any/all") do
+    @fact any(identity, list(true, false, false)) --> true
```

Comment:
This is ambiguous on 0.5 (but curiously not on 0.4 or 0.6). Any idea how best to address that?